Framework/MDAlgorithms/test/ConvertMDHistoToMatrixWorkspaceTest.h

```diff
@@ -494,4 +494,101 @@ class ConvertMDHistoToMatrixWorkspaceTest : public CxxTest::TestSuite {
   }
 };
 
+/* Disabling clang format because the CXX parsser cannot read across multiple
+lines.
+ Clang formatting has therefore been disabled to prevent compiler errors.
+ When the parsser has been fixed (can read across multiple lines) the clang
+formatting
+can be enabled */
+
+// clang-format off
+class ConvertMDHistoToMatrixWorkspaceTestPerformance:public CxxTest::TestSuite {
+public:
+  static ConvertMDHistoToMatrixWorkspaceTestPerformance *createSuite() {
+    return new ConvertMDHistoToMatrixWorkspaceTestPerformance;
+  }
+  static void destroySuite(ConvertMDHistoToMatrixWorkspaceTestPerformance *suite) {
+    delete suite;
+  }
+//clang-format on
+  ConvertMDHistoToMatrixWorkspaceTestPerformance() {
```

Comment:
This is all setup code - if you change this to
`void setup(){ ...}`
And move the following test setup code into that method we no longer have to create this suite statically.